app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt

```diff
@@ -33,20 +37,19 @@ fun IntroScreen(onGetStartedClicked: () -> Unit) {
         horizontalAlignment = Alignment.CenterHorizontally,
         verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.dimension_16dp))
```

Comment:
Could you please make the column scrollable? On my device I cannot see button because of screen aspect ratio. Also be careful here on spaceBy and Spacer - Spacer is a composable component and will also be influenced by spaceBy, for instance, if the structure is component A -> Spacer(height 16dp) -> component B, the Spacer generates 16(spaceBy component A) + 16(its height) + 16(spaceBy component B) = 48dp of space

Reply:
Made the changes